internal/scorecard/scorecard.go

```diff
@@ -181,6 +180,7 @@ func initConfig() error {
 		}
 		log.Info("Using config file: ", viper.ConfigFileUsed())
 	} else {
+		//TODO: What to change here
```

Comment:
No need to change anything here in this PR.

Reply:
Removed the TODO comment.